src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -386,6 +391,11 @@
           "type": "string"
         },
 
+        "decoratedName": {
+          "description": "The fully qualified name for the logical location, in the compiler's native format.",
```

Comment:
I'm ok with all that. I'd like to describe the FQLN as "human-readable" and the decoratedName as "machine-readable". I'm a little uncomfortable with "non-readable".

Also I'm sure someone will take offense at "mangled" but I'm prepared to defend it ;-).

---
In reply to: [63254920](https://github.com/Microsoft/sarif-sdk/pull/272#discussion_r63254920) [](ancestors = 63254920,63254879)